script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5.txt

```diff
@@ -4,22 +4,22 @@ I run through the crowded streets.
 The fire in front of the station can be seen even from here.
 <Page2>
 The screens outside broadcast the breaking news on repeat.
-A fire had broken out in the vicinity of Souya Station's north exit.
+A fire had broken out in the vicinity of Souya Station's north gate.
 The trains are suspended and the people who were visiting the downtown area are frantically running for their lives.
 <Page3>
-A news anchor is reporting the emergency news on the outdoor screens. //text overflow, the entire page needs rephrasing 
+A news anchor is reporting the emergency on the outdoor screens.//text overflow, the entire page needs rephrasing
 It's an interview with a high school girl who has just escaped the fire.
 "It's hot, it's hot. Hey, why is it so hot?
-My friend, Yuka, just got eaten by them. They even got my arm. It's like...
+My friend, Yuka, was eaten by them. They even got my arm. It's like...
```

Comment:
I left "just got" because it sounded more frantic like you'd hear a shocked person on the news say. revert imo